tests/travis/install_sqlanywhere.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+wget -qO- "https://drive.google.com/uc?export=download&id=$SQLANYWHERE_GDRIVE_ID" | tar xz # SQL Anywhere server
```

Comment:
@Ocramius well doesn't have to be me but that's the only option to enable builds that don't require a long setup time. Basically what I did is:

- Download server package from SAP
- Install with developer license key (which you get after registration)
- Remove unnecessary files to shrink size
- tar folder
- upload to GDrive

"Maintenance" is a bit of an exaggeration here IMO because the package won't change. It simply works...

Better ideas are always welcome :)